2-mandatory/2-water-bottle.js

```diff
@@ -19,20 +19,36 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    return (this.volume = 100);
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    // for (let i = this.volume; i < 100; i += 10) {
+    //   this.volume += i;
+    //   if (this.volume === 100) {
+    //     break;
+    //   }
+    // }
+
+    this.volume += 10;
+    this.volume > 100 ? (this.volume = 100) : this.volume;
   },
   drink: function () {
     // calling this function should decrease your bottle volume by 10 units;
+    this.volume -= 10;
+    this.volume < 0 ? (this.volume = 0) : this.volume;
   },
   isFull: function () {
     // this function should return true if your bottle is full;
+    return this.volume === 100;
   },
   isEmpty: function () {
     // this function should return true if your bottle is empty;
+    return this.volume === 0;
   },
 };
+// bottle.fillUp();
+// console.log(bottle.volume);
```

Comment:
Good, this works well and is clear and concise. I think it is fine for you to keep commented out code in these exercises as it shows your process in finding a solution, but in a real life situation you would normally remove commented out code unless it is informative.